app/controllers/fae/nested_base_controller.rb

```diff
@@ -85,7 +85,7 @@ def raise_undefined_parent
     end
 
     def table_template_path
-      "#{fae.root_path.gsub('/', '')}/#{@klass_name}/table"
+      "#{fae.root_path.gsub('/', '')}admin/#{@klass_name}/table"
```

Comment:
Doesn't it need a `/` before admin? How example path looks like?

Reply:
e.g. `app/views/admin/certificates/table.html.slim`
It seems that render works both with and without prefixing the path with `/`.